openqaoa/problems/problem.py

```diff
@@ -71,7 +71,7 @@ class QUBO:
     # Maximum number of terms allowed to enable the cleaning procedure
     TERMS_CLEANING_LIMIT = 5000
 
-    def __init__(self, n, terms, weights, clean_terms_and_weights=False):
+    def __init__(self, n, terms, weights, clean_terms_and_weights=False, constant=0):
```

Comment:
good idea to add `constant` argument here. Let's bring it before the `clean_terms_and_weights` argument?

Reply:
Yeah, that's a good point.

We need to check if we have something like `qubo(n, terms, weights, False)` which may crash if we do

Reply:
Nothing crashed in the tests! Let's hope for the best!